exchanges/bybit/bybit_wrapper.go

```diff
@@ -0,0 +1,1844 @@
+	"github.com/thrasher-corp/gocryptotrader/exchanges/kline"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/orderbook"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/protocol"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/stream"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/stream/buffer"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/trade"
+	"github.com/thrasher-corp/gocryptotrader/log"
+	"github.com/thrasher-corp/gocryptotrader/portfolio/withdraw"
+)
+
+// GetDefaultConfig returns a default exchange config
+func (by *Bybit) GetDefaultConfig() (*config.Exchange, error) {
+	by.SetDefaults()
+	exchCfg := new(config.Exchange)
+	exchCfg.Name = by.Name
+	exchCfg.HTTPTimeout = exchange.DefaultHTTPTimeout
+	exchCfg.BaseCurrencies = by.BaseCurrencies
+
+	err := by.SetupDefaults(exchCfg)
+	if err != nil {
+		return nil, err
+	}
+
+	if by.Features.Supports.RESTCapabilities.AutoPairUpdates {
+		err := by.UpdateTradablePairs(context.TODO(), true)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return exchCfg, nil
+}
+
+// SetDefaults sets the basic defaults for Bybit
+func (by *Bybit) SetDefaults() {
+	by.Name = "Bybit"
+	by.Enabled = true
+	by.Verbose = true
+	by.API.CredentialsValidator.RequiresKey = true
+	by.API.CredentialsValidator.RequiresSecret = true
+
+	requestFmt := &currency.PairFormat{Uppercase: true}
+
+	configFmt := &currency.PairFormat{Delimiter: currency.DashDelimiter, Uppercase: true}
+	err := by.SetGlobalPairsManager(requestFmt, configFmt, asset.Spot, asset.CoinMarginedFutures, asset.USDTMarginedFutures, asset.Futures, asset.USDCMarginedFutures)
+	if err != nil {
+		log.Errorln(log.ExchangeSys, err)
+	}
+
+	err = by.DisableAssetWebsocketSupport(asset.CoinMarginedFutures)
+	if err != nil {
+		log.Errorln(log.ExchangeSys, err)
+	}
+
+	err = by.DisableAssetWebsocketSupport(asset.USDTMarginedFutures)
+	if err != nil {
+		log.Errorln(log.ExchangeSys, err)
+	}
+
+	err = by.DisableAssetWebsocketSupport(asset.Futures)
+	if err != nil {
+		log.Errorln(log.ExchangeSys, err)
+	}
+
+	err = by.DisableAssetWebsocketSupport(asset.USDCMarginedFutures)
+	if err != nil {
+		log.Errorln(log.ExchangeSys, err)
+	}
+	by.Features = exchange.Features{
+		Supports: exchange.FeaturesSupported{
+			REST:      true,
+			Websocket: true,
+			RESTCapabilities: protocol.Features{
+				TickerFetching:    true,
+				TradeFetching:     true,
+				KlineFetching:     true,
+				OrderbookFetching: true,
+				AutoPairUpdates:   true,
+				AccountInfo:       true,
+				GetOrder:          true,
+				GetOrders:         true,
+				CancelOrders:      true,
+				CancelOrder:       true,
+				SubmitOrder:       true,
+				DepositHistory:    true,
+				WithdrawalHistory: true,
+				UserTradeHistory:  true,
+				CryptoDeposit:     true,
```

Comment:
These were technically false, but now they support deposit/withdrawal endpoints and once completed, can be left as true. Once the account asset deposits/withdrawals are implemented, can also set `MultiChainDeposits` and `MultiChainWithdrawals` to true